src/pybind/mgr/volumes/fs/fs_util.py

```diff
@@ -214,3 +214,22 @@ def create_base_dir(fs, path, mode):
             fs.mkdirs(path, mode)
         else:
             raise VolumeException(-e.args[0], e.args[1])
+
+
+def validate_earmark(self, earmark):
```

Comment:
As discussed in the original downstream design doc, I'd like to see this function in a more common location. For example something like `src/python-common/ceph/fs/earmarking.py`.

Reply:
Done!